lib/util.js

```diff
@@ -51,10 +51,15 @@ function getLevelDOWN () {
   try {
     return leveldown = require('leveldown')
   } catch (e) {
-    throw new LevelUPError(missingLevelDOWNError)
+    throwRequireError(e)
   }
 }
 
+function throwRequireError (e) {
+  var template = 'Failed to require LevelDOWN (%s). Try `npm install leveldown` if it\'s missing'
+  throw new LevelUPError(format(template, e.message))
```

Comment:
IMO the `throw` should still be outside the function, ie it should only return the error